feat(blog): a32nx flight model (#477)
flybywiresim/website: src/posts/a32nx-flight-model.md

---
title: 'A32NX Flight Model'
date: 'tbd'
authors:
  - 'donbike'
category: 'UPDATE'
metaImage: '/img/notam-images/flypados3/flypad-hero.png'
metaAlt: 'FlyByWire Simulations'
---

## Overview

> We have gone to great lengths to ensure that the A32NX flight model is as realistic as it gets. Collaborating with our own A320 pilot staff at FlyByWire Simulations and testing each change religiously the flight model is based on matching the following:

Author: Excellent feedback. I'll work in the changes this weekend. 

As for the charts I'm still considering best way to style them but I agree with altering some from raw data to visual such as bar graph in the case of stopping distances.

---
title: 'A32NX Flight Model'
date: '2022-12-23'
authors:
  - 'donbikes'
  - 'Valastiri'
category: 'UPDATE'
metaImage: '/img/flight-model-feature.png'
metaAlt: 'FlyByWire Simulations'
---

## Overview

We have gone to great lengths to ensure that the A32NX flight model is as realistic as it gets. Collaborating with our own A320 pilot staff at FlyByWire Simulations and testing each change 

> meticulously, the flight model is based on matching the data and other information obtained from the A320NEO Airplane Flight Manual, Flight Crew Operating Manual, Flight Crew Techniques Manual, 

Reviewer: ```suggestion
meticulously, we have adjusted the flight model to data and other information obtained from the A320NEO Airplane Flight Manual, Flight Crew Operating Manual, Flight Crew Techniques Manual, 
```

Quick Reference Handbook, FlySmart electronic flight bag, and our A320NEO piloting team.

> If you are a more data oriented reader, below you can find comparisons between the reference data and our tests in MSFS. We have included a plethora of tables and charts for you.

Reviewer: ```suggestion
If you are a more data-oriented reader, below you can find comparisons between the reference data and our tests in MSFS. We have included a plethora of tables and charts for you.
```

Credits:

- donbikes for the flight model work,
- Taz for the engine modelling,
- Davy for the simulation of the autobrake system.

> **Note:** Some of the smaller more data heavy images can be clicked on so you can view the data in full.

Reviewer: ```suggestion
**Note:** Some of the smaller, more data-heavy images can be clicked on so you can view the data in full.
```

---

## Work In Progress Tuning

> Constant improvements with realistic and often very recent data points from our pilot staff is where we believe the A32NX shines. Recent updates to our aircraft fixes issues with inadequate 

Reviewer: ```suggestion
Constant improvement with realistic and often very recent data points from our pilot staff is where we believe the A32NX shines. Recent updates to our aircraft fix issues with inadequate 
```

takeoff climb performance after an engine failure at heavy gross weights. You can now practice those V1 engine cuts at 2nd segment climb-limited TOGW. Be sure when doing so to not delay gear 
retraction, lower your pitch rate and pitch target, and apply appropriate rudder to minimize control and sideslip drag! (**Note:** This capability is currently only available in the Development 
and Experimental versions of the A32NX).

We are currently fine-tuning the landing gear drag to better match climb, cruise, and descent data for gear down operations. The results of our WIP tuning are included in the tables below. We 

> expect to add these changes to the A32NX soon - first in the Experimental version, followed by the Development version, and then our Stable releases.

Reviewer 2: I would not phrase it like this as this gives the impression that Experimental is the first release where we publish new stuff. This is not the case!
We do testing of selected feature in Exp - which is basically Dev +selection of to-be-tested features

```suggestion
expect to add these changes to the A32NX soon - we will start testing these in the Experimental version, then once they are sufficiently tested and stable we will publish it to the Development version, and eventually to our our Stable releases.
```

Author: great point -- slight issue with ur edit (duplicate "our") -- will commit manually in a bit. Thanks!
